src/psyclone/tests/psyir/transformations/matmul2code_trans_test.py

```diff
@@ -0,0 +1,480 @@
+# -----------------------------------------------------------------------------
```

Comment:
This file has pylint errors with trailing whitespace.

Reply:
Fixed errors. Again sorry.